source/renderer/app/containers/staking/dialogs/redeem-itn-rewards/Step1ConfigurationContainer.js

```diff
@@ -17,6 +17,7 @@ export default class Step1ConfigurationContainer extends Component<Props> {
 
   render() {
     const { onClose, onBack, stores, actions } = this.props;
+    const { isSynced, syncPercentage } = stores.networkStatus;
```

Comment:
It makes more sense to have this logic inside **RedeemItnRewardsContainer.js**, as it already has the **NoWalletsContainer.js**

Reply:
@daniloprates I do not agree. If we do that then it could happen that sync progress drops below 100% in the middle of the redemption process in which case we would revert to that "Not in sync" step and that would be wrong.